build/azure-pipelines/cli/vcpkg-deps.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  - name: targets
+    default: []
+    type: object
+  - name: vcpkgDir
+    type: string
+  - name: targetDirectory
+    type: string
+
+steps:
+  - script: git clone https://github.com/microsoft/vcpkg.git $(Build.ArtifactStagingDirectory)/vcpkg
```

Comment:
Are we really cloning this bindly? What if this repo gets compromised?

Reply:
I'll have this check out the latest commit on the repo to provide some assurance here.